joomla/joomla-cms
# [5.2] Fix filtering exclude and include articles

### Summary of Changes
The new article model has a article id filtering, it was a simple text field where you could add ids. This is changed to a subform with a article modal to chose articles. Further more it is not possible to exclude and include articles by id at the same time. 

I have changed the configuration of the module so that is only allow what is possible.

#### Exclude Mode  
![02A9B0B4-0C64-41F5-B220-08A8EF8F9B00](https://github.com/user-attachments/assets/91376aae-2965-4809-a2d6-05ecd45d96ac)
#### Include Mode
![4317BFBF-3F9F-42AE-8707-688B8A3AAC8C](https://github.com/user-attachments/assets/aec34ce1-a0f9-4c92-a0d1-cd6b387af97c)

### Testing Instructions
Try to exclude some articles and include some articles. Check if the module show the expected articles. 

### Expected result AFTER applying this Pull Request
Module shows the expected articles

### Link to documentations
Please select:
